mlir/test/Target/Cpp/verbatim.mlir

```diff
@@ -40,6 +40,18 @@ emitc.func @func(%arg: f32) {
 
   emitc.verbatim "#pragma my2 var={} property" %a : !emitc.array<3x7xi32>
   // CHECK-NEXT: #pragma my2 var=[[A]] property
+  emitc.return
+}
 
+// Check that no semicolon is printed after verbatim within emitc.for
+emitc.func @in_loop(%arg: f32) {
+  %start = emitc.literal "0" : !emitc.size_t
+  %stop = emitc.literal "10" : !emitc.size_t
+  %step = emitc.literal "1" : !emitc.size_t
+  emitc.for %iter = %start to %stop step %step {
+    emitc.verbatim "#pragma"
+     // CHECK: #pragma
```

Comment:
`CHECK` does not check the full line, right? Won't you need a
```
CHECK-NOT: ;
CHECK: Whatever the next thing is
```
to rule out that there is a `;` at the end?

Reply:
This test case uses `FileCheck %s --match-full-lines` (see top of file). I have verified that this test case fails before the fix.

Reply:
Ah sorry I didn't notice that. Thanks!